src/librustdoc/html/render.rs

```diff
@@ -4345,19 +4345,21 @@ fn sidebar_trait(buf: &mut Buffer, it: &clean::Item, t: &clean::Trait) {
             .iter()
             .filter(|i| i.inner_impl().for_.def_id().map_or(false, |d| !c.paths.contains_key(&d)))
             .filter_map(|i| match extract_for_impl_name(&i.impl_item) {
-                Some((ref name, ref id)) => {
-                    Some(format!("<a href=\"#{}\">{}</a>", id, Escape(name)))
-                }
+                Some((name, id)) => Some((id, name)),
```

Comment:
Do we really need to swap the order in this tuple? In fact if we leave it as `(name, id)` then we can continue to just use `res.sort();` below.

Reply:
I didn't pay attention, good catch!